tests/chain_plugin_tests.cpp

```diff
@@ -194,7 +198,20 @@ BOOST_AUTO_TEST_CASE( get_consensus_parameters ) try {
    BOOST_TEST(parms.wasm_config->max_code_bytes == t.control->get_global_properties().wasm_configuration.max_code_bytes);
    BOOST_TEST(parms.wasm_config->max_pages == t.control->get_global_properties().wasm_configuration.max_pages);
    BOOST_TEST(parms.wasm_config->max_call_depth == t.control->get_global_properties().wasm_configuration.max_call_depth);
+   BOOST_TEST(!parms.chain_config.get_object().contains("max_sync_call_depth"));
+   BOOST_TEST(!parms.chain_config.get_object().contains("max_sync_call_data_size"));
 
+   // verifying max_sync_call_depth and max_sync_call_data_size
+   t.preactivate_builtin_protocol_features( {builtin_protocol_feature_t::sync_call} );
+   t.produce_block();
+
+   parms = plugin.get_consensus_parameters({}, fc::time_point::maximum());
+   chain_config_v2 v2config;
+   from_variant(parms.chain_config, v2config);
+
+   BOOST_TEST(v2config.max_action_return_value_size == t.control->get_global_properties().configuration.max_action_return_value_size);
```

Comment:
This line looks like it might be a copy/paste from above -- don't you want to check that `max_sync_call_depth` and `max_sync_call_data_size` match -- not `max_action_return_value_size`?

Reply:
Thanks! I should have verified the returned values of max_sync_call_depth and max_sync_call_data_size

https://github.com/AntelopeIO/spring/pull/1331/commits/28b5f9f164700a0fa0e14f2d3888c55dad30f41b